### DISCUSSION TITLE
Any chance this gets integrated with PowerToys?

### DISCUSSION THREAD

**Discussion by Viridel** [kangyu-california/PersistentWindows#166] [Ideas]
Seems like a great match, since PT is all about fixing what Windows has screwed up, which is the definition of why this utility exists :)

**Comment by kangyu-california** [comment]
The main issue that PW is trying to solve has been addressed in Windows 11, I guess PowerToys may not have the incentive to refix the issue.

Technically the integration is possible, but it will not be a easy one ( and I wish Microsoft to initiate the integration)
PowerToys (including FancyZones) is mainly written in C++ and is licensed under MIT, while PW is written in C# and licensed under GPL3.

But you have made a good point, which reminded me that PowerToys might already have solved some technical problem which PW is painfully searching a solution for.

Thanks for the suggestion.
